react/index.native.js

```diff
@@ -3,7 +3,7 @@
 // Apply all necessary polyfills as early as possible to make sure anything imported henceforth
 // sees them.
 import './features/mobile/polyfills';
-
+import 'react-native-gesture-handler';
```

Comment:
Is it necessary to import this early? If so, please leave a comment explaining, with some link to docs.

Reply:
Checked again and no. In the documentation it was mentioned that we need it, but everything seems to work normally without it.